<launch>
  <node name="haam_nodes" pkg="haam_ros_pkg" type="haam_nodes.py" output="screen">
    <param name="device_ip" value="10.10.10.1" />
    <param name="detection_mode" value="2" />
    <param name="log_data" value="0" />
  </node>
</launch>